### DISCUSSION TITLE
Is it possible to launch seleniumbase in UC mode in google colab?

### DISCUSSION THREAD

**Discussion by go-to-mjrecent-on-TG** [seleniumbase/SeleniumBase#3330] [Q&A] [answered]
https://stackoverflow.com/questions/79175347/use-seleniumbase-in-uc-mode-in-google-colab

**Comment by mdmintz** [comment, marked as answer]
You may want to see the existing discussions: https://github.com/search?q=repo%3Aseleniumbase%2FSeleniumBase+colab&type=discussions

In your link, it said: `ModuleNotFoundError: No module named 'pyautogui'`.  You should make sure that is installed first. (Or check permissions, because SeleniumBase will install that automatically at runtime if there are the permissions to do so.)

**Reply by mdmintz** [reply]
Someone from the [SeleniumBase Discord server](https://discord.gg/EdhQTn3EyE) got it working on Google Colab with:

```python
from seleniumbase import SB
import subprocess
import nest_asyncio
nest_asyncio.apply()

# Start an Xvfb session
subprocess.run(['xvfb-run', '-a'])

from seleniumbase import SB

with SB(uc=True, test=True, headed=True, locale_code="en") as sb:
    sb.activate_cdp_mode(URL)
    # ...
```
